components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -104,6 +104,8 @@
     public static final String IDP_ID = "idpId";
     public static final String ASSOCIATED_ID = "associatedID";
 
+    public static final String PROVISIONED_USER = "provisionedUser";
```

Comment:
Don't we need to say that this is JIT provisioned user?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3589/commits/7f93dadc8d2fd9e9edaeb3ce334c1c627c083ca6